### DISCUSSION TITLE
Comparisons with match length

### DISCUSSION THREAD

**Discussion by BrownBearSec** [projectdiscovery/httpx#1077] [Q&A] [answered]
### Please describe your feature request:

the ability for the `-ml` flag to be able to do comparisons as well as single numbers, eg `-ml >100` as well as `-ml 100`

### Describe the use case of this feature:

This is generally just a good feature, but in my case, it would be good for detecting larger files like backups etc.

**Comment by ehsandeep** [comment, marked as answer]
@iamNynan as shared in discord, this is already supported using DSL based mater / filter, for example -

```
echo example.com | httpx -mdc 'content_length > 10 && content_length < 10000' -cl

    __    __  __       _  __
   / /_  / /_/ /_____ | |/ /
  / __ \/ __/ __/ __ \|   /
 / / / / /_/ /_/ /_/ /   |
/_/ /_/\__/\__/ .___/_/|_|
             /_/

        projectdiscovery.io

[INF] Current httpx version v1.2.9 (latest)
https://example.com [1256]
```
